packages/cookie-handler/src/index.js

```diff
@@ -0,0 +1,55 @@
+import { features } from '@goldinteractive/js-base'
+import invariant from 'tiny-invariant'
+
+import Cookies from 'js-cookie'
+
+class CookieHandler extends features.Feature {
+  init() {
+    this.cookieIdentifier =
+      this.node.dataset.cookieIdentifier || this.options.cookieIdentifier
+    if (this.cookieIdentifier == null) {
+      invariant(false, 'Cookie identifier needs to be set.')
```

Comment:
The idea behind the first param is to omit the surrounding if construct.
`invariant(this.cookieIdentifier !== null, 'Cookie identifier needs to be set.')`